src/Card.css

```diff
@@ -0,0 +1,11 @@
+.card {
+  display: flex;
+  background: #f2f2f2;
+  border-radius: 25px;
+  padding: 20px;
+  width: 100%;
+  max-width: 600px;
+  height: 700px;
```

Comment:
this should be a flexible unit (`%`, `rem`, `em`) to make it dynamic or responsive, you could add a `min-height` or `max-height` to limit the size